### ISSUE TITLE
[Feature request] Make inputs relative to cursor

### ISSUE THREAD

**Issue by abdullah-if** [H-M-H/Weylus#98]
Currently inputs act absolute to the screen, moves the cursor where I put the finger on video. It will be great if a 
 option is added so that the input was relative to current position of the cursor. Can this feature added ?

**Comment by H-M-H** [comment]
What is your use case? To me screen mirroring with relative input seems counterintuitive. The only thing I can think of is using smaller devices like phones as touch pad with mirroring disabled, perhaps this is worth some consideration.

**Comment by ar-nelson** [comment]
+1 to this: it doesn't make sense with mirroring, but would be very useful when using Weylus as an ordinary drawing pad. Whenever I use a drawing pad with my 4K monitor, I always have it in relative mode, because the screen is too large to use absolute cursor position comfortably.

**Comment by abdullah-if** [comment]
@H-M-H yes. In case of smartphone with small display, I can use it as touchpad.

**Comment by gabrielmaster123** [comment]
for me it is the opposite way, and I want it to be absolute how do I change that

**Comment by mosaleh52** [comment]
This will change my workflow 
before discovering this great app I was dependent on iPad with 'goodnotes 5' for taking notes which something I was not comfortable with, currently I'm using xournal++ with iPad which works very good except for the ergonomics. 
with this feature I can use xournal++ with iPad as ordinary graphic tablet  saving my neck 
+ could be helpful for those whom want to use their phone as trackpad
+ for designer that needs more accurate devices this is essential
 'this is  my point of view with title experience with illustration software and extensive digital note-taking experience '

**Comment by gabmert** [comment]
touchpad mode is something issue #158 asks for

**Comment by SotchNam** [comment]
bump
